packages/e2e-tests/specs/a11y.test.js

```diff
@@ -29,6 +30,48 @@ describe( 'a11y', () => {
 		expect( isFocusedToggle ).toBe( true );
 	} );
 
+	it( 'checks persistent selection', async () => {
+		await clickBlockAppender();
+		await page.keyboard.type( 'Testing editor selection persistence' );
+
+		// moving focus backwards using keyboard shortcuts
+		// twice to get to the inspector tabs
+		await pressKeyWithModifier( 'ctrlShift', '`' );
+		await pressKeyWithModifier( 'ctrlShift', '`' );
+
+		await page.keyboard.press( 'Tab' );
+
+		const isFocusedInspectorDocumentTab = await page.$eval( ':focus', ( focusedElement ) => {
+			return focusedElement.getAttribute( 'data-label' );
+		} );
+
+		expect( isFocusedInspectorDocumentTab ).toEqual( 'Document' );
+
+		await page.keyboard.press( 'Space' );
+
+		let isFocusedParagraphBlock = await page.$eval( ':focus', ( focusedElement ) => {
```

Comment:
To be precise, this checks whether RichText component is focused rather than any block. In case of Paragraph block it happens to be the only component used.

Reply:
I don't really understand what you mean :)

Reply:
This doesn't check whether Paragraph block is focused as the name implies: `isFocusedParagraphBlock`. It checks whether RichText component is focused. There are plenty of blocks which use RichText component(s) so this is confusing.

Reply:
updated the wording and the flow to match what is really happening.